src/deque.jl

```diff
@@ -148,12 +144,10 @@ function Base.iterate(qi::ReverseDequeIterator{T}, (cb, i) = (qi.q.rear, qi.q.re
     return (x, (cb, i))
 end
 
-reverse_iter(q::Deque{T}) where {T} = ReverseDequeIterator{T}(q)
-
 Base.iterate(q::Deque{T}, s...) where {T} = iterate(DequeIterator{T}(q), s...)
 
 Base.length(qi::DequeIterator{T}) where {T} = qi.q.len
-Base.length(qi::ReverseDequeIterator{T}) where {T} = qi.q.len
+Base.length(qi::Iterators.Reverse{<:Deque}) = qi.itr.len
```

Comment:
I think we can delete this line, as it is defined in `Base`
https://github.com/JuliaLang/julia/blob/5f0469b2273c8de57a26aec88468e5b5eed3a61b/base/iterators.jl#L101

Reply:
```suggestion
```